x/recovery/keeper/keeper.go

```diff
@@ -31,20 +37,19 @@ type Keeper struct {
 
 // NewKeeper returns keeper
 func NewKeeper(
-	ps paramtypes.Subspace,
+	storeKey storetypes.StoreKey,
+	cdc codec.BinaryCodec,
+	authority sdk.AccAddress,
 	ak types.AccountKeeper,
 	bk types.BankKeeper,
 	ck types.ChannelKeeper,
 	tk types.TransferKeeper,
 	claimsKeeper types.ClaimsKeeper,
 ) *Keeper {
-	// set KeyTable if it has not already been set
-	if !ps.HasKeyTable() {
-		ps = ps.WithKeyTable(types.ParamKeyTable())
-	}
-
 	return &Keeper{
-		paramstore:     ps,
+		storeKey:       storeKey,
+		cdc:            cdc,
+		authority:      authority,
```

Comment:
@Vvaradinov we should validate that this is a valid address (i.e not `nil` or empty) before setting it to the keeper